Library/Homebrew/cask/cask_loader.rb

```diff
@@ -45,7 +45,10 @@ def load(config:)
       private
 
       def cask(header_token, **options, &block)
-        Cask.new(header_token, source: content, **options, config: @config, &block)
+        checksum = {
+          "sha256" => Digest::SHA256.hexdigest(content),
+        }
+        Cask.new(header_token, source: content, source_checksum: checksum, **options, config: @config, &block)
```

Comment:
Should this argument be made mandatory if things fail when it's not specified?

Reply:
It is not mandatory if `sourcefile_path` is passed.

I don't really like the million parameters that `Cask`'s constructor can take so will see if there's anything better.

Reply:
Ah, gotcha, thanks.